City-of-Pori/pori-kada
# PORI-172 fixes to sections

To test:

1. drush fra -y && drush cc all
2. Make sure section front page doesn't show all the feed tabs if all of them are not chosen. Make sure only the once chosen are shown.
3. Make sure the section menu is shown correctly in a news content that is attached to the section.
4. Make sure there is News archive link on the section front page and it is in correct place, make sure news archive link is small and above news title in news item that is attached to a section.
5. Make sure main image is not displayed on basic page that is attached to a section.
